src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.cs

```diff
@@ -73,8 +74,14 @@ DkmCompiledClrLocalsQuery IDkmClrExpressionCompiler.GetClrLocalVariableQuery(
                         builder.Free();
                         return new GetLocalsResult(typeName, locals, assembly);
                     },
-                    out error);
-                return DkmCompiledClrLocalsQuery.Create(runtimeInstance, null, CompilerId, r.Assembly, r.TypeName, r.Locals);
+                    out var result,
+                    out var error))
+                {
+                    return DkmCompiledClrLocalsQuery.Create(runtimeInstance, null, CompilerId, result.Assembly, result.TypeName, result.Locals);
+                }
+
+                // TODO: better error reporting to the debugger
```

Comment:
>TODO [](http://example.com/codeflow?start=19&length=4)

Let's remove "TODO". If we think we need to improve the error reporting, please log an issue and use that issue link instead of TODO. Otherwise, simply remove the comment.